{"uuid": "d6ff9121-2a93-4a3b-9bda-72a5b58ce34d", "children": ["89e55dab-09fa-4382-b672-8c52da83dcd5", "f2547316-3ab2-4152-b168-0b6219812e8b", "667cabc5-4134-4fee-8e87-59559a3458e7", "5e861433-26db-43d1-b85b-2bd402cc9822", "f52785b3-4823-4966-947f-8aae486f5642", "3c31d3bc-9434-44ee-9bab-02f609207abe"], "befores": [{"name": "setup_profiles", "status": "broken", "statusDetails": {"message": "TypeError: 'bool' object is not iterable\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/wlan-testing/tests/e2e/basic/conftest.py\", line 79, in setup_profiles\n    return_var = fixtures_ver.setup_profiles(request, param, setup_controller, testbed, get_equipment_id,\n  File \"/wlan-testing/tests/fixtures_2x.py\", line 292, in setup_profiles\n    ap_wifi_data = ap_ssh.get_interface_details()\n  File \"/wlan-testing/tests/../libs/apnos/apnos.py\", line 396, in get_interface_details\n    for i in r:\n"}, "attachments": [{"name": "ucentral_config: ", "source": "a582622a-0de5-4e3e-9f3a-ce96af47d685-attachment.attach"}, {"name": "Time Took to apply Config: 230.50490140914917", "source": "028434ee-2d7f-40ad-b75b-1824caeca1c2-attachment.attach"}, {"name": "Success : Active Config in AP: ", "source": "ac2b8b0d-2429-47f7-a474-7750b781183c-attachment.attach"}, {"name": "AP LOgs: ", "source": "aa679e2c-9e58-4f4f-84f9-6d4b41d31433-attachment.attach"}], "start": 1629079429623, "stop": 1629079833626}], "start": 1629079429623, "stop": 1629079834068}